src/main/xerus/monstercat/downloader/TabDownloader.kt

```diff
@@ -53,7 +53,7 @@ import java.util.concurrent.TimeUnit
 import kotlin.math.absoluteValue
 import xerus.ktutil.javafx.ui.controls.Type as SearchType
 
-private val qualities = arrayOf("mp3_128", "mp3_v2", "mp3_v0", "mp3_320", "flac", "wav")
```

Comment:
They removed MP3 variable bitrate? What the heck?

Reply:
yes. Only MP3_320, FLAC and WAV are available anymore. They used to redirect the other MP3 qualities but they don't anymore